Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -23,6 +24,27 @@ public class TeleportPointer : LinePointer, IMixedRealityTeleportPointer, IMixed
         [Tooltip("The threshold amount for joystick input (Dead Zone)")]
         private float inputThreshold = 0.5f;
 
+        private float InputThreshold
```

Comment:
Does this get used / referenced anywhere? Or is it meant for future-proofing? I don't think this can currently change at runtime

Reply:
it can if running in the editor....

Reply:
though future-proofing is also a nice side effect :)

Reply:
Changing the value in the inspector won't call this code path though